Tests/SnapshotTests/Extensions/XCTestCase+Extensions.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  Created by Backbase R&D B.V. on 07/12/2022.
+//
+
+import XCTest
+import SnapshotTesting
+import BackbaseDesignSystem
+import SwiftUI
+
+extension XCTestCase {
+
+    // MARK: - Snapshot testing helper methods
+
+    /// Asserts and verifies given view controller
+    ///  - Parameters:
+    ///     - viewController: view controller to verify
+    ///     - waitTime: waiting time before timing out while verifying an image
+    ///     - file: File name of the snapshot test class
+    ///     - function: A unique method name of the snapshot test to be used to attach to the name of the image
+    ///     - named: Optional name to attach to the snapshot test. By default it uses the name of the method from `function` parameter.
+    ///  - Returns: A list of error descriptons in case existing image did not match with the created image.
+    func verifyViewControllerSnapshot(with viewController: UIViewController,
```

Comment:
we should probably remove this function as we dont use it, or we should call `verifyViewSnapshot` from within it, as i can see there is some inconsistencies between this function and the other one, probably should do the same in MDS

Reply:
Done, removed!